Src/Common/Domain/implementation/VolunteerOpportunity.cs

```diff
@@ -8,42 +9,60 @@ public class VolunteerOpportunity : IVolunteerOpportunity
         #region Constructors
 
         public VolunteerOpportunity()
-            : this(null)
+            : this(null, null)
         {
         }
 
-        public VolunteerOpportunity(IUser createdBy)
+        public VolunteerOpportunity(IUser createdBy, IOrganization organization)
         {
             CreatedBy = createdBy;
+            Organization = organization;
 
             RecordId = 0;
-            StartTime = DateTime.Today;
-            StopTime = DateTime.Today;
+
+            ProjectName = string.Empty;
+            ProjectDescription = string.Empty;
+
+            StartTime = DateTime.MinValue;

```

Comment:
Why the switch from Today to MinValue? If you start with Today, the Opporunity would start with a date that is somewhat relevant. Starting with MinValue can actually cause an issue in some SQL Servers (like Microsoft SQL Server has an issue with at DateTime < 1/1/1900) when using a normal DateTime datatype.